yt/funcs.py

```diff
@@ -774,38 +703,6 @@ def bb_apicall(endpoint, data, use_pass=True):
     return urllib.request.urlopen(req).read()
 
 
-def get_yt_supp():
-    import hglib
-
-    supp_path = os.path.join(os.environ["YT_DEST"], "src", "yt-supplemental")
-    # Now we check that the supplemental repository is checked out.
-    if not os.path.isdir(supp_path):
-        print()
-        print("*** The yt-supplemental repository is not checked ***")
-        print("*** out.  I can do this for you, but because this ***")
-        print("*** is a delicate act, I require you to respond   ***")
-        print("*** to the prompt with the word 'yes'.            ***")
-        print()
-        response = input("Do you want me to try to check it out? ")
-        if response != "yes":
-            print()
-            print("Okay, I understand.  You can check it out yourself.")
-            print("This command will do it:")
-            print()
-            print(
-                "$ hg clone http://bitbucket.org/yt_analysis/yt-supplemental/ ", end=" "
-            )
-            print(f"{supp_path}")
-            print()
-            sys.exit(1)
-        rv = hglib.clone("http://bitbucket.org/yt_analysis/yt-supplemental/", supp_path)
-        if rv:
-            print("Something has gone wrong.  Quitting.")
-            sys.exit(1)
-    # Now we think we have our supplemental repository.
-    return supp_path
```

Comment:
I know this repo isn't carried in the github project org, but I'm curious what the legacy is from it @matthewturk or @brittonsmith?

Reply:
OK, that takes me back.  I had to look it up.  Last commit I see was 2012, but `yt-supplemental` was a set of subrepos that included some things that didn't quite belong in yt proper.

```
yt-doc = http://hg.enzotools.org/yt-doc
hgbb = http://BitBucket.org/MatthewTurk/hgbb
pasteboard_template = http://BitBucket.org/MatthewTurk/pasteboard_template
cedit = http://bitbucket.org/MatthewTurk/cedit
yt-advanced-plotting = http://bitbucket.org/samskillman/plot_modifications
```

So it was my fork of the `hgbb` extension to make it easier to manage bitbucket (kind of like a really lightweight `hub` or `gh` command), a `pasteboard` for sharing scripts (never totally worked but it was fun), the docs, a configuration editing extension for hg, and @samskillman 's advanced plotting code that fiddled with plotting settings.

Reply:
Very cool!!! Maybe we should revisit some of these things (like the advanced plotting stuff)? 

So it seems like it's very likely this isn't used by anybody? Or should this be in a deprecation cycle instead? Or do we even want to remove it?

Reply:
the urls are dead atm, which is why I'm proposing to delete this. Of course if the material is still available elsewhere (and still compatible), it should probably be kept and updated instead.

Reply:
I totally support deletion! I didn't realize they were dead -- and it seems likely that this is unused (I didn't see any uses outside of forks) so I think it makes sense totally remove. It might be cool later on to revisit adding in "advanced" modules  in a formalized way for the project.